api/swagger-spec/v1.json

```diff
@@ -11703,6 +11703,9 @@
    },
    "v1.NodeStatus": {
     "id": "v1.NodeStatus",
+    "required": [
+     "daemonPorts"
```

Comment:
Yes, but this PR is built on top of #12655, so you needn't worry about the first commit.